### PR TITLE
Add support for adis16475 driver

### PR DESCRIPTION
**Pull request by rbolboac**
# Description

Add support for adis16475 driver

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Hardware:
* OS:

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -254,6 +254,7 @@ def checkemulation(c):
         "adf5610",
         "adg2128",
         "adis16460",
+        "adis16475",
```
Comment (tfcollins) [inline tasks.py:257]: We don't allow new parts to be added without emulation support , unless for very specific reasons. Can you provide a context file for the system (see here https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#new-hardware-requirements and https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#emulation)?

**Review by tfcollins — CHANGES_REQUESTED** [review]
See comment about emulation support

File: tasks.py (hunk 1 above)
Comment (rbolboac) [reply tasks.py:257]: Done

Comment (tfcollins) [reply tasks.py:257]: Please remove from this list.

File: adi/adis16475.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        "accel_z",
+        "temp0",
+    ]
+    _device_name = ""
+
+    def __init__(self, uri="", device_name="adis16505-2"):
+        context_manager.__init__(self, uri, self._device_name)
+
+        compatible_parts = [
+            "adis16470",
```
Comment (tfcollins) [inline adi/adis16475.py:28]: Please add this list to the supported parts md file.

File: test/emu/hardware_map.yml [hunk 3]
```diff
@@ -397,3 +397,12 @@ ad4858:
       - data_devices:
           - iio:device0
           - iio:device2
+
+adis16475:
+  - adis16475
+  - emulate:
+      - filename: adis16475.xml
```
Comment (tfcollins) [inline test/emu/hardware_map.yml:404]: This file is missing from the test/emu/devices folder

**Review by tfcollins — CHANGES_REQUESTED** [review]
Missing emu context file and need some doc updates.

**Comment by rbolboac** [comment]
> Missing emu context file and need some doc updates.

I apologize, I skipped the xml file in the commit.

File: adi/adis16475.py (hunk 2 above)
Comment (rbolboac) [reply adi/adis16475.py:28]: Should I do anything else besides adding ADIS16475 to the list?

**Review by tfcollins — APPROVED** [review]

**Pull request merged**